wiki/osu!wiki_Contribution_Guide/Common_Issues/id.md

```diff
@@ -0,0 +1,80 @@
+# Masalah Umum
+
+*Halaman utama: [Panduan Kontribusi osu!wiki](/wiki/owcg).*
```

Comment:
`osu!wiki` -> `osu! wiki`
Can you do that for every instance? Including the name of the file. A recent PR was pushed with these minor changes.

Reply:
Will do when i have time, 👍